src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -102,7 +106,16 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
   });
 
   // note: this is here just to change the type on it in an easy way
-  const enterRoomWithSound: () => void = _enterRoomWithSound;
+  const enterRoomWithSound: () => void = useCallback(() => {
+    _enterRoomWithSound();
+    if (venue.template === VenueTemplate.partymap && isExternalUrl(room.url)) {
```

Comment:
add a not on why you have an if there, please

Reply:
fixed in 3cc41543e7a8e5a46377a4027fd29f05fa07d811